ui/src/resources/reducers/helpers.ts

```diff
@@ -46,12 +39,9 @@ export const editResource = <R>(
   resource: ResourceType
 ) => {
   const {entities, result} = action.schema
-  if (!draftState.byID) {
-    draftState.byID = {}
-  }
 
-  const bucket = entities[resource][result]
-  draftState.byID[result] = bucket
+  const r = entities[resource][result]
```

Comment:
why have a transient variable called `r`? `r` doesn't provide any more meaning than `entities[resource][result]` what do you think about changing this to:

```ts
draftState.byID[result] = entities[resource][result]
```
?

Reply:
its certainly better than `buckets`.  it's really a resource but resource is already taken 🤷‍♂.  Good enough for golang!

Reply:
the point i'm making is that it's already clear from the assignment and the name `entities[resource][result]` that `r` represents a result, and if we can eliminate one more line of code without losing any meaning, i'm generally 110% for that.

>Good enough for golang!

we're not writing go though, we're writing typescript / javascript